cmake/scripts/ROOTConfig.cmake.in

```diff
@@ -130,9 +130,13 @@ if(ROOT_minuit2_omp_FOUND)
 endif()
 if(ROOT_vdt_FOUND AND NOT TARGET VDT::VDT)
   if(ROOT_builtin_vdt_FOUND)
-    set(Vdt_ROOT "${_thisdir}/.." CACHE PATH "Location of (the builtin) VDT")
+    function(find_builtin_vdt)
+      # the function is to create a scope (could use block() but requires CMake>=3.25)
+      set(CMAKE_PREFIX_PATH ${ROOT_INCLUDE_DIRS} ${ROOT_LIBRARY_DIR})
+      find_dependency(Vdt)
```

Comment:
Sorry for the delay, I was on holiday last week...

Are `ROOT_INCLUDE_DIRS` and `ROOT_LIBRARY_DIR` really needed/appropriate here? If we are executing this code, that means that CMake managed to find `ROOTConfig.cmake`. At that point, wouldn't the include and library directories also be found by CMake? 🤔

But even more than this, does this additional scope not hide the `VDT::VDT` target as well? Unless it's created as a `GLOBAL` target, will it be visible outside of the function as well? I honestly don't know...

Reply:
> Sorry for the delay, I was on holiday last week...

No problem don't worry!

And frankly speaking I don't know, but since it seems to fix issues on MacOS...